### PR TITLE
Using public getter to retrieve nonce instead of hardcoding

### LINKED ISSUE #1741: DI demo-code: use public getter to retrieve nonce instead of hardcoding
### Context
Using `PublicGetter` impl in https://github.com/litentry/litentry-parachain/pull/1729.

- [ ] modify the demo-code of DI to use this getter to retrieve nonce instead of hardcoding
- [ ] CI error: https://github.com/litentry/litentry-parachain/commit/302e2f5211043f381edcef8c1c9f93dfe783dd60

### LINKED ISSUE #1745: Failing tee-test lit-identity-direct-invocation-test
We have got failing CI, example:
https://github.com/litentry/litentry-parachain/actions/runs/5111858175/jobs/9190401064

After a short investigation it seems that the return value of user_shielding_key trusted getter has changed

prev:
`{ value: '0x00', do_watch: false, status: 'Ok' }`
actual:
`{ value: '0x010400', do_watch: false, status: 'Ok' }`

this makes test fail on:
`assert.isTrue(k.isNone, 'shielding key should be empty before set');`

### PR DESCRIPTION
**Pull request by outofxxx**
fix: #1741
fix: #1745

### CODE REVIEW

File: tee-worker/ts-tests/examples/direct-invocation/util.ts [hunk 1]
```diff
@@ -260,5 +294,15 @@ export const createRequest = async (
     } else {
         cyphertext = compactAddLength(bufferToU8a(encryptWithTeeShieldingKey(teeShieldingKey, top)));
     }
+
     return parachain_api.createType('Request', { shard: hexToU8a(mrenclave), cyphertext }).toU8a();
 };
+
+// decode nonce
+export function decodeNonce (nonceInHex: any) {
```
Comment (kziemianek) [inline tee-worker/ts-tests/examples/direct-invocation/util.ts]: I think you are expecting `string` here, can we use strict type ?

**Review by kziemianek — APPROVED** [review]
Please run `npm run format`

**Review by kziemianek — APPROVED** [review]
Please run `npm run format`

File: tee-worker/ts-tests/examples/direct-invocation/util.ts (hunk 1 above)
Comment (outofxxx) [reply tee-worker/ts-tests/examples/direct-invocation/util.ts]: yes

**Pull request merged**